rollup/pipeline/pipeline.go

```diff
@@ -327,9 +332,6 @@ func (p *Pipeline) cccStage(candidates <-chan *BlockCandidate, deadline time.Tim
 					}
 					return
 				}
-				deadlineReached = true
-				// avoid deadline case being triggered again and again
-				deadline = time.Now().Add(time.Hour)
```

Comment:
This is indeed not necessary as the timer's channel is not closed when it fired. If it was closed it would trigger again and again (see [here with a closed channel](https://go.dev/play/p/rOjdvnji49) and [here with a timer](https://go.dev/play/p/sIzrGs4h16d)).